Userland/Applications/Maps/MapWidget.cpp

```diff
@@ -171,6 +174,43 @@ void MapWidget::mousewheel_event(GUI::MouseEvent& event)
     set_zoom_for_mouse_event(new_zoom, event);
 }
 
+void MapWidget::context_menu_event(GUI::ContextMenuEvent& event)
+{
+    if (!m_context_menu_enabled)
+        return;
+
+    LatLng latlng = { tile_y_to_latitude(latitude_to_tile_y(m_center.latitude, m_zoom) + static_cast<double>(event.position().y() - height() / 2) / TILE_SIZE, m_zoom),
```

Comment:
Nitpick, but why is this `LatLong` class called `LatLng`? We don't have to be afraid of vowels here :)

Reply:
That seems to be what google uses as a name?
<https://developers.google.com/maps/documentation/javascript/examples/map-latlng-literal>
I personally prefer lat/lon, but aperantly all three ways are used

Reply:
You are right I used the name convetions of other map libraries like Leaflet.js and the Google Maps API. I would suggest for now to keep them the same, in the future we can always change it off course.